Source/Core/VideoBackends/D3D/DXTexture.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "VideoCommon/AbstractTexture.h"
+#include "VideoCommon/VideoCommon.h"
+
+class D3DTexture2D;
+
+namespace DX11
+{
+
+class DXTexture : public AbstractTexture
+{
+public:
+
+  D3DTexture2D* GetRawTexIdentifier() const { return texture; }
```

Comment:
This should have its implementation in the cpp file. This should also be placed after the virtual functions.